api/test/wfl/integration/firecloud_test.clj

```diff
@@ -34,11 +35,7 @@
    #(let [get-workflows (fn [] (:workflows (firecloud/get-submission workspace %)))
           workflow      (first (get-workflows))]
       (is (= (second entity) (get-in workflow [:workflowEntity :entityName])))
-      (is (#{"Queued" "Submitted"}
-           ;; GH-1212: `get-submission` can return the workflow before it has
-           ;; been queued. In such cases :status is nil so try again
-           (or (:status workflow)
-               (util/poll (comp :status first get-workflows))))))))
+      (is (not (str/blank? (util/poll (comp :status first get-workflows))))))))
```

Comment:
So now we are just checking that the first workflow has a status defined?

Reply:
yep. That's as much as we care about really.